config/locales/two_factor_authentication/en.yml

```diff
@@ -20,7 +20,10 @@ en:
     header_text: Enter your one-time code
     important_alert_icon: important alert icon
     invalid_backup_code: That backup code is invalid.
-    invalid_otp: That one-time code is invalid. You can try entering it again or
+    invalid_otp_html: 
+      one: That one-time code is invalid. You can try entering it again or request a new one-time code. You have <strong>%{count} attempt</strong> remaining.
+      other: That one-time code is invalid. You can try entering it again or request a new one-time code.
```

Comment:
Unsure if it's what you were referring to in your other comment about showing all attempts left, but should we show this sooner than on the very last attempt? It seems like could be abrupt for the user to be suddenly informed that they're on their final attempt. In document capture, we [display it on the last three attempts](https://github.com/18F/identity-idp/blob/7abed4f6aa2b32b53b356d5ba25535c673cba0b1/app/javascript/packages/document-capture/components/review-issues-step.tsx#L56).

We could have separate strings for "invalid without attempts remaining shown" and then "invalid with few attempts remaining (one or other)".

Reply:
Yes thats what I was referring to. I think I will update to contain multiple attempts.